include/ignition/gui/qml/GzColor.qml

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+import QtQuick 2.9
+import QtQuick.Controls 2.1
+import QtQuick.Dialogs 1.0
+import QtQuick.Layouts 1.3
+
+
+GridLayout {
+  id: root
+  property string colorName
+
+  property alias r: r_spin.value
+  property alias g: g_spin.value
+  property alias b: b_spin.value
+  property alias a: a_spin.value
+
+  property bool textVisible: false
```

Comment:
nit: `textVisible` isn't very descriptive. Can we rename this to `showColorText`?

Reply:
Remove `textVisible` in 8835c72